asset-transfer-basic/rest-api-typescript/src/server.ts

```diff
@@ -13,13 +13,19 @@ import { healthRouter } from './health.router';
 import { jobsRouter } from './jobs.router';
 import { logger } from './logger';
 import { transactionsRouter } from './transactions.router';
+import cors from 'cors';
+import { devCorsEnabled } from './config';
 
 const { BAD_REQUEST, INTERNAL_SERVER_ERROR, NOT_FOUND } = StatusCodes;
 
 export const createServer = async (): Promise<Application> => {
   const app = express();
 
   // Remember for production usage, to check any TLS or CORS requirements
+  // this wildcard option *MIGHT* not be applicable to your environment
+  if (devCorsEnabled) {
```

Comment:
Following on from config comment, I think it would be good to move this to the existing dev mode block

```
if (process.env.NODE_ENV === 'development') {
  // TBC
}
```

https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-basic/rest-api-typescript/src/server.ts#L54

Reply:
So...

```
  if (process.env.NODE_ENV === 'development') {
    app.use(cors());
  }
```